src/buildfromsource/FSharp.Build/AssemblyInfo.fs

```diff
@@ -8,7 +8,7 @@ open System.Reflection
 [<assembly:AssemblyTitle("FSharp.Build.dll")>]
 [<assembly:AssemblyCopyright("\169 Microsoft Corporation.  All Rights Reserved.")>]
 [<assembly:AssemblyProduct("Microsoft\174 F#")>]
-[<assembly:AssemblyInformationalVersion("10.1.1.0")>]
-[<assembly:AssemblyVersion("10.1.1.0")>]
-[<assembly:AssemblyFileVersion("2017.06.27.0")>]
+[<assembly:AssemblyInformationalVersion("10.2.0.0")>]
```

Comment:
Unrelated to this PR, but we (I) should probably update `buildfromsource/` to parameterize these values via the project files since they should be building against a copy of `FSharp.Build.dll` that has `WriteCodeFragment`.

Reply:
@brettfo yes we should, that occured to me, while doing this.  I think we have all of the bits in place now.